components/Vote/ElectionDiversity.js

```diff
@@ -155,7 +155,11 @@ const MembersGender = voteT(({ members, vt }) => {
             ? { ...item, value: item.value + 1 }
             : item
         ),
-      Object.keys(GENDER).map(key => ({ key, value: 0 }))
+      Object.keys(GENDER).map(key => ({
+        key,
+        value: 0,
+        pos: key === 'weiblich' ? 'left' : 'right'
```

Comment:
wouldn't it be more logical to be index based? index `0` left rest right?

Reply:
Right you are: there you go.